starlette_exporter/__init__.py

```diff
@@ -22,8 +24,19 @@
 from .middleware import PrometheusMiddleware
 from .labels import from_header
 
+class MetricsHandler:
+    def __init__(self, registry: CollectorRegistry = REGISTRY):
+        """A class wrapper that allows you to inject your own metrics registry instead of the default."""
+        self.registry = registry
+
+    def handle_metrics(self, request: Request) -> Response:
+        return handle_metrics(request, self.registry)
+
+    def handle_openmetrics(self, request: Request) -> Response:
+        return handle_openmetrics(request, self.registry)
+
 
-def handle_metrics(request: Request) -> Response:
+def handle_metrics(request: Request, registry: Optional[CollectorRegistry] = None) -> Response:
```

Comment:
Would this allow passing an arbitrary value via a `registry=foo` URL query param? If so, this method of extending the existing handler is probably best avoided. Maybe one option is to leave this simple handler alone, and the class based handling can be for situations where more configuration is needed? Thoughts?